### DISCUSSION TITLE
readback/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#35] [Announcements]
# readback/

And Now For Something Slightly More Technical

https://www.supergoodcode.com/readback/

**Comment by amitbar05** [comment]
Great work again :)
So does this solution introduce 1 frame input latency on those specific applications?

**Reply by zmike** [reply]
No, it eliminates the latency that previously existed. FPS may be slightly reduced as compared to identical rendering that didn't do swapchain readback, but that's unavoidable.

**Reply by jams3223** [reply]
But does it offer more stable frames for some lower FPS?
